cactus/plugin/builtin/pagecontext.py

```diff
@@ -0,0 +1,120 @@
+#coding:utf-8
```

Comment:
Can you name this file `page_context.py`? This would be more consistent with e.g. `template_tags.py ` and generally how Python files are called